go/vt/vttablet/tabletserver/query_engine_test.go

```diff
@@ -188,18 +188,20 @@ func TestQueryPlanCache(t *testing.T) {
 	ctx := context.Background()
 	logStats := tabletenv.NewLogStats(ctx, "GetPlanStats")
 	if cache.DefaultConfig.LFU {
-		qe.SetQueryPlanCacheCap(1024)
+		// this cache capacity is in bytes
```

Comment:
nice commenting. it's a good habit, adding comments for things that made you scratch your head :)